sdk/core/azure-core/src/test/java/com/azure/core/models/PollResultTests.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models;
+
+import com.azure.core.util.BinaryData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PollResultTests {
+
+    @Test
+    public void testDeserialization() {
+        String successResponseBody = "{\"id\":\"Uw_N1zoT53-rhHeaB5UJ_BaWKv8QVanU_Rjmo5FVdRs\",\"status\":\"Succeeded\",\"error\":null}";
```

Comment:
NIT:  worth to create an static val for `Uw_N1zoT53-rhHeaB5UJ_BaWKv8QVanU_Rjmo5FVdRs`

Reply:
it's fine as-is

Reply:
Thanks Shawn for the suggestion. I've extracted the 2 operationId in assertion (but keep the 2 in JSON).